lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -689,6 +689,18 @@ def wrap_in_middleware( app, global_conf, **local_conf ):
         from galaxy.web.framework.middleware.translogger import TransLogger
         app = TransLogger( app )
         log.debug( "Enabling 'trans logger' middleware" )
+    # Statsd request timing and profiling
+    statsd_host = conf.get('statsd_host', None)
+    if statsd_host:
+        try:
```

Comment:
I think it would be better if Galaxy just died if a configured optional feature is required but the library is unavailable.